invenio_search_ui/views.py

```diff
@@ -177,8 +202,10 @@ def aggs(self):
         return [
             {
                 "title": k.capitalize(),
-                "aggName": k,
-                "field": v["terms"]["field"]
+                "agg": {
```

Comment:
Moderate: We will need to also pass the ```childAgg``` here. On another note, I feel like the nesting in the aggregation to ```agg``` might create some confusion, since alternatively we will have just the redundant ```title```.

Reply:
This is a very good point. However, I feel that the point here is to have a very basic instance running (which does not have nested facets configured). If someone were to configure said facets, wouldn't it also need to overwrite how this is passed?

About naming, I agree, it was already confusing for us...

Reply:
As discussed we can leave the ```childAgg``` for now and provide a simpler solution since there is already the capability to overwrite the ```aggs``` in the template and provide it.

Reply:
Ok from me to leave out ``childAgg``

Reply:
We're supporting `childAgg` in our current CERN Open Data `master` branch to allow sub-facets. In case you're interested in our implementation:

https://github.com/cernopendata/opendata.cern.ch/blob/5ade73621b29d8a45b9f682d4206d2a9831b99c3/cernopendata/modules/search_ui/helpers.py#L36-L65